net-mail/gnubiff/Changelog

```diff
@@ -0,0 +1,11 @@
+# Changelog for net-mail/gnubiff
```

Comment:
We don't do these anymore -- could you show me which document told you to do it? (It's fine if you just made a mistake but I'm wondering if there's some old doc giving bad instructions.)

Reply:
I think I got the habit from looking at other ebuilds.  If we no longer do that, I'll make a note to stop creating them.

Reply:
You shouldn't see any ChangeLogs in ::gentoo since like 2015 :)